# Task Action ROS Message Definition

string type
geometry_msgs/Pose pose
trajectory_msgs/JointTrajectory traj